Add Caldor Fire dataset (#143)
NASA-IMPACT/delta-config: datasets/caldor-fire-characteristics-burn-severity.data.mdx

---
id: caldor-fire-behavior-burn-severity
name: "Caldor Fire Behavior and Burn Severity"
description: "Assets describing the progression and active fire behavior of the 2021 Caldor Fire in California."
media:
  src: ::file ./caldor-fire--dataset-cover.jpg
  alt: Hillside engulfed by a wildfire.
  author:
    name: Marek Piwnicki
    url: https://unsplash.com/photos/WiZOyYqzUss
thematics:
  - eis
layers:
  - id: caldor-fire-behavior
    stacCol: caldor-fire-behavior
    name: Fire Behavior
    type: raster
    description: ""
    initialDatetime: newest
    zoomExtent:
      - 10
      - 20
    sourceParams:
      colormap_name: inferno_r
      rescale:
        - -41

>         - 98

Reviewer: The new files apparently have a different min max value? (ref: the image in this PR conversation).
I reran the ingestion pipeline - so the min max value have changed:
check here: https://staging-stac.delta-backend.com/collections/caldor-fire-behavior/items

>         - -41
>         - 98

Reviewer: ```suggestion
        - 0
        - 93
```

Reviewer 2: @slesaad thank you for making all these changes! I have one other note related to the data scaling: the burn severity raster only has valid values from 1-4. Is it possible to change those values accordingly? I also saw your note on whether `inferno` or `inferno_r` would be better... honestly I think either work, but your comment about warmer colors indicating higher severity values makes sense to me. 

Last thing: is it possible to replace "Fire Behavior" with "Mean Fire Radiative Power (MW)"?

    legend:
      type: gradient
      min: "-41"
      max: "98"
      stops:
        - "#FBB41A"
        - "#BB3754"
        - "#781D6D"
        - "#34095F"
  - id: caldor-fire-burn-severity
    stacCol: caldor-fire-burn-severity
    name: Burn Severity
    type: raster
    description: "Soil burn severity (SBS) conditions."
    initialDatetime: newest
    zoomExtent:
      - 10
      - 20
    sourceParams:
      colormap_name: inferno_r
      rescale:
        - 0
        - 5
    legend:
      type: gradient
      min: "0"
      max: "5"
      stops:
        - "#FBB41A"
        - "#BB3754"
        - "#781D6D"
        - "#34095F"
  
---

<Block>
<Prose>
  ## About

>   This dataset describes the progression and active fire behavior of the 2021 Caldor Fire in California, analyzed via the algorithm detailed in https://www.nature.com/articles/s41597-022-01343-0. It includes an extra layer detailing the soil burn severity (SBS) conditions provided by the [Burned Area Emergency Response](https://burnseverity.cr.usgs.gov/baer/) team.

Reviewer 3: @eorland Take a look at this text.

Reviewer 2: Text looks good! I would _maybe_ change the words "analyzed via" to the phrase "as recorded by" but this really is me splitting hairs.

>   This dataset describes the progression and active fire behavior of the 2021 Caldor Fire in California, analyzed via the algorithm detailed in https://www.nature.com/articles/s41597-022-01343-0. It includes an extra layer detailing the soil burn severity (SBS) conditions provided by the [Burned Area Emergency Response](https://burnseverity.cr.usgs.gov/baer/) team.

Reviewer: ```suggestion
  This dataset describes the progression and active fire behavior of the 2021 Caldor Fire in California, as recorded by the algorithm detailed in https://www.nature.com/articles/s41597-022-01343-0. It includes an extra layer detailing the soil burn severity (SBS) conditions provided by the [Burned Area Emergency Response](https://burnseverity.cr.usgs.gov/baer/) team.
```

>   `.geojson` and `tif` files describing the progression and active fire behavior of the 2021 Caldor Fire in California via the algorithm detailed in https://www.nature.com/articles/s41597-022-01343-0. This includes an extra `.tif` file detailing the soil burn severity (SBS) conditions provided by the [Burned Area Emergency Response](https://burnseverity.cr.usgs.gov/baer/) team.

Reviewer 4: ```suggestion
This dataset describes the progression and active fire behavior of the 2021 Caldor Fire in California, analyzed via the algorithm detailed in https://www.nature.com/articles/s41597-022-01343-0. It includes an extra layer detailing the soil burn severity (SBS) conditions provided by the [Burned Area Emergency Response](https://burnseverity.cr.usgs.gov/baer/) team.
```
